common/assets/scss/core/_elements.scss

```diff
@@ -10,3 +10,7 @@ img {
   display: inline-block;
   max-width: 100%;
 }
+
+// [id] {
+//   scroll-margin-top: 100px; // Approximate height of sticky navigation.
```

Comment:
This appears to be causing TestCafe to have issues selecting nested elements, need to investigate.